tests/Microsoft.Azure.Batch.SoftwareEntitlement.Tests/EntitlementVerifierTests.cs

```diff
@@ -14,37 +14,37 @@
 
 namespace Microsoft.Azure.Batch.SoftwareEntitlement.Tests
 {
-    public class TokenEnforcementTests
+    public class EntitlementVerifierTests
     {
-        // A valid software entitlement to use for testing
-        private readonly NodeEntitlements _validEntitlements;
+        // An entitlements object representing a complete set of claims
+        private readonly NodeEntitlements _completeEntitlement;
+
+        // An entitlement verification request which is valid for the above entitlement
+        private readonly EntitlementVerificationRequest _validEntitlementRequest;
 
         // Generator used to create a token
         private readonly TokenGenerator _generator;
 
         // Verifier used to check the token
-        private readonly TokenVerifier _verifier;
+        private readonly EntitlementVerifier _verifier;
 
         // Current time - captured as a member so it doesn't change during a test
         private readonly DateTimeOffset _now = DateTimeOffset.Now;
 
         // A application identifiers for testing
-        private readonly string _contosoFinanceApp = "contosofinance";
-        private readonly string _contosoITApp = "contosoit";
-        private readonly string _contosoHRApp = "contosohr";
+        private readonly string _approvedApp = "contosofinance";
 
         // IP addresses to use
-        private readonly IPAddress _otherAddress = IPAddress.Parse("203.0.113.42");
         private readonly IPAddress _approvedAddress = IPAddress.Parse("203.0.113.45");
 
         // Name for the approved entitlement
         private readonly string _entitlementIdentifer = "mystery-identifier";
 
         // Audience to which tokens should be addressed
-        private readonly string _audience = "https://account.region.batch.azure.test";
+        private readonly string _audience = "https://audience.region.batch.azure.test";
```

Comment:
URL is intended to mirror the real URL used for production batch, which has the form `https://account.region.batch.azure.text`

Reply:
Changed to flag any errors in the unit test that mixed up `_audience` and  `_issuer`; agreed to leave.